### PR TITLE
Initialize echopype conda recipe

### PR DESCRIPTION
**Pull request by lsetiawan**
## Overview

This PR Initialize the echopype recipe.

> echopype is a package built for enhancing the interoperability and scalability in ocean sonar data processing. These data are widely used for obtaining information about the distribution and abundance of marine animals, such as fish and krill. Our ability to collect large volumes of sonar data from a variety of ocean platforms has grown significantly in the last decade. However, most of the new data remain under-utilized. echopype aims to address the root cause of this problem - the lack of interoperable data format and scalable analysis workflows that adapt well with increasing data volume - by providing open-source tools as entry points for scientists to make discovery using these new data.

Pinging @leewujung.

@conda-forge/staged-recipes Please review and merge this recipe, if everything looks good. Thank you very much!

### CODE REVIEW

**Comment by lsetiawan** [comment]
Pinging @ocefpaf. This package allows an open source way in working with acoustic data. Thought you would want to know, in case IOOS wants to utilize it :smile:.

File: recipes/echopype/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  license_family: APACHE
+  license_file: LICENSE
+  summary: Enhancing interoperability and scalability in analyzing water column sonar data.
+  doc_url: https://echopype.readthedocs.io/en/latest/
+  dev_url: https://github.com/OSOceanAcoustics/echopype
+
+extra:
+  recipe-maintainers:
+    - lsetiawan
+    - leewujung
```
Comment (scopatz) [inline recipes/echopype/meta.yaml]: @leewujung are you OK with being a maintainer?

Comment (leewujung) [reply recipes/echopype/meta.yaml]: Yes sounds good. Thanks!

**Comment by scopatz** [comment]
noarch python

**Comment by lsetiawan** [comment]
Thank you @scopatz for being so quick on approving this recipe! :smile:

**Comment by ocefpaf** [comment]
> Pinging @ocefpaf. This package allows an open source way in working with acoustic data. Thought you would want to know, in case IOOS wants to utilize it .

Thanks!

**Pull request merged**